drivers/can/can_mcp2515.c

```diff
@@ -461,11 +461,11 @@ static void mcp2515_handle_interrupts(struct device *dev)
 			mcp2515_tx_done(dev, 0);
 		}
 
-		if (canintf & MCP2515_CANINTF_TX0IF) {
+		if (canintf & MCP2515_CANINTF_TX1IF) {
```

Comment:
Just curious; don't we need RX2 as well?

Reply:
@ioannisg no the chip has 3 TX buffers and 2 RX buffers.